src/interface/swarm.hpp

```diff
@@ -66,6 +66,7 @@ class Swarm {
 
  public:
   Swarm(const std::string &label, const Metadata &metadata, const int nmax_pool_in = 3);
+  Swarm() : Swarm("default", Metadata()) {}
```

Comment:
Where is this needed?

Reply:
Not sure it's needed anymore. I will try removing it.

Reply:
Does not appear to be required. Will remove.